components/style/properties/shorthand/mask.mako.rs

```diff
@@ -218,9 +221,14 @@
             }
 
             for i in 0..len {
-                self.mask_position_x.0[i].to_css(dest)?;
-                dest.write_str(" ")?;
-                self.mask_position_y.0[i].to_css(dest)?;
+                let position_x = &self.mask_position_x.0[i];
+                let position_y = &self.mask_position_y.0[i];
+
+                Position {
+                    horizontal: position_x.clone(),
+                    vertical: position_y.clone()
```

Comment:
It may be more compact to
```rust
Position {
    horizontal: self.mask_position_x.0[i].clone(),
    vertical: self.mask_position_y.0[i].clone(),
}
```